lib/std/coff.zig

```diff
@@ -794,9 +794,36 @@ pub const MachineType = enum(u16) {
     }
 };
 
-const IMAGE_NUMBEROF_DIRECTORY_ENTRIES = 16;
-const IMAGE_DEBUG_TYPE_CODEVIEW = 2;
-const DEBUG_DIRECTORY = 6;
+pub const IMAGE_NUMBEROF_DIRECTORY_ENTRIES = 16;
+
+pub const IMAGE_DEBUG_TYPE = struct {
```

Comment:
While you're making this change, would you mind converting this into an integer-backed enum? I think it would make more sense than a struct with consts.

Reply:
I really wanted to, but in the past we've explicitly avoided them for the various flags in `std.os` so I followed suit here. I don't recall the exact reasoning why, though.

Reply:
It looks like we're now using a number of them in various places in `std.os`, so I guess it's okay now? Will revise.

Reply:
Thanks! And yeah, I think using an integer-backed enum for flags, and packed struct for bitsets is the way to go. It cleans up a lot of code IMHO. This change will also aid me in my quest of writing an incremental COFF linker.